lib/src/wallet.rs

```diff
@@ -1558,6 +1747,18 @@ impl LightWallet {
                     .unwrap();
                 spent_note.unconfirmed_spent = Some((transaction.txid(), u32::from(target_height)));
             }
+            // Mark orchard notes as unconfirmed spent
```

Comment:
Okay, lets DRY it!

Reply:
https://github.com/zingolabs/zingolib/issues/114#issuecomment-1260171623

Tracked --^